rustfmt-core/rustfmt-bin/src/bin/main.rs

```diff
@@ -483,11 +483,18 @@ fn format(opt: Opt) -> Result<i32> {
         return Err(format_err!("Error: `{}` is a directory", dir.display()));
     }
 
-    let (default_config, config_path) = load_config(None, Some(&opt))?;
+    let (default_config, config_paths) = load_config(None, Some(&opt))?;
 
     if opt.verbose {
-        if let Some(path) = config_path.as_ref() {
-            println!("Using rustfmt config file {}", path.display());
+        if let Some(paths) = config_paths.as_ref() {
+            println!(
+                "Using rustfmt config files {}",
```

Comment:
Just in case there's only one file
```suggestion
                "Using rustfmt config file(s) {}",
```